panther_analysis_tool/backend/client.py

```diff
@@ -156,21 +156,33 @@ def bulk_upload(self, params: BulkUploadParams) -> BackendResponse[BulkUploadRes
         pass
 
     @abstractmethod
-    def delete_saved_queries(self, params: DeleteSavedQueriesParams) -> BackendResponse[DeleteSavedQueriesResponse]:
+    def delete_saved_queries(
+            self, params: DeleteSavedQueriesParams
+    ) -> BackendResponse[DeleteSavedQueriesResponse]:
         pass
 
     @abstractmethod
-    def delete_detections(self, params: DeleteDetectionsParams) -> BackendResponse[DeleteDetectionsResponse]:
+    def delete_detections(
+            self, params: DeleteDetectionsParams
+    ) -> BackendResponse[DeleteDetectionsResponse]:
         pass
 
     @abstractmethod
-    def list_managed_schemas(self, params: ListSchemasParams) -> BackendResponse[ListManagedSchemasResponse]:
+    def list_managed_schemas(
+            self, params: ListSchemasParams
+    ) -> BackendResponse[ListManagedSchemasResponse]:
         pass
 
     @abstractmethod
     def update_managed_schema(self, params: UpdateManagedSchemaParams) -> BackendResponse[Any]:
         pass
 
     @abstractmethod
-    def panthersdk_bulk_upload(self, params: PantherSDKBulkUploadParams) -> BackendResponse[PantherSDKBulkUploadResponse]:
+    def panthersdk_bulk_upload(
+            self, params: PantherSDKBulkUploadParams
+    ) -> BackendResponse[PantherSDKBulkUploadResponse]:
         pass
+
+
+def backend_response_failed(resp: Union[BackendResponse, dict]) -> bool:
+    return resp.status_code >= 400 or resp.data["statusCode"] >= 400  # type: ignore
```

Comment:
I'm a little confused about this method. Can you clarify why it accepts `resp: Union[BackendResponse, dict]`? Does the method really work if passed a dict?

Reply:
Sorry it can't. I never cleaned this up from when I was developing it